net/netns/netns_linux.go

```diff
@@ -26,19 +28,50 @@ import (
 // wgengine/router/router_linux.go.
 const tailscaleBypassMark = 0x80000
 
-// ipRuleOnce is the sync.Once & cached value for ipRuleAvailable.
-var ipRuleOnce struct {
+// socketMarkWorksOnce is the sync.Once & cached value for useSocketMark.
+var socketMarkWorksOnce struct {
 	sync.Once
 	v bool
 }
 
-// ipRuleAvailable reports whether the 'ip rule' command works.
+// socketMarkWorks returns whether SO_MARK works. If unsure, it returns true as the
```

Comment:
can you add `// unsure; assume yes` comments on the unsure exit paths?

Reply:
Done.